network-api/networkapi/buyersguide/templates/buyersguide/product_page.html

```diff
@@ -331,7 +331,8 @@ <h3 class="h2-heading mb-4">{% trans "Related products" %}</h3>
   {% if coralTalkServerUrl %}
     <script src="{{ coralTalkServerUrl }}static/embed.js" async onload="
       Coral.Talk.render(document.getElementById('coral-talk-stream'), {
-        talk: '{{ coralTalkServerUrl }}'
+        talk: '{{ coralTalkServerUrl }}'{% if lang_code == "en" %},
+        asset_url: '{{ product.get_parent.url }}products/{{ product.slug }}/'{% endif %}
```

Comment:
based on Coral's admin interface, existing threads point to `https://foundation.mozilla.org/en/privacynotincluded/products/nintendo-switch/`, so this templating code effects the same URL (although localhost testing will obviously not resolve to foundation.mozilla.org...)